tests/Resource/ResourceServicesTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Resource;
+
+use Doctrine\ORM\EntityManager;
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+final class ResourceServicesTest extends WebTestCase
+{
+    /**
+     * @test
+     */
+    public function it_allows_to_access_resource_services_from_container(): void
+    {
+        $client = parent::createClient();
+
+        $productRepository = $client->getContainer()->get('sylius.repository.product');
```

Comment:
Can we make this tests ResourceBundle specific? We don't need the whole Sylius app to test public services.